### DISCUSSION TITLE
I can't install Hubs Cloud on AWS. - API: s3:CreateBucket Access Denied

### DISCUSSION THREAD

**Discussion by jwatabe** [mozilla/hubs#5425] [Hubs Cloud Questions]
I can't install Hubs Cloud on AWS.
I'm following the steps to set up Hubs Cloud, but I get the following error when creating a stack in CloudFormation.
the steps：https://github.com/gree/hubs-docs-jp/wiki/hubs-cloud-aws-quick-start
(Hubs CloudをAWSにインストールできません。
Hubs Cloudのセットアップを手順通りに進めていますが、CloudFormationでスタック作成時に以下のようなエラーが発生します。
手順：https://github.com/gree/hubs-docs-jp/wiki/hubs-cloud-aws-quick-start)
<img width="1009" alt="スクリーンショット 2022-05-12 11 07 44" src="https://user-images.githubusercontent.com/100191650/167983241-60a5d3c9-4c18-4da9-96ca-3bbe9b2c7146.png">
<img width="1009" alt="スクリーンショット 2022-05-12 11 07 02" src="https://user-images.githubusercontent.com/100191650/167978048-92473c35-e9a9-458f-b1c0-0c1df57f052d.png">
Here is the first error that occurred in the event.
(イベントで発生した最初のエラーはこちらです。)
<img width="1009" alt="スクリーンショット 2022-05-12 11 06 20" src="https://user-images.githubusercontent.com/100191650/167978186-4d2bf3f3-ed30-444a-81ba-ef51d32cf6e1.png">
Hubs Cloud products choose personal.
I'm trying to create a region in northern Virginia.
The stack parameters were set as follows.
mysite.com and mysite.link are the domains I got with Route53. 
Leave the other parameters at their defaults.
(Hubs Cloud製品はpersonalを選んでいます。
リージョンはバージニア北部で作成しようとしています。
スタックのパラメータは以下のように設定しました。
mysite.comとmysite.linkは、Route53で取得したドメインです。
その他のパラメータはデフォルトのままにしています。)
<img width="1009" alt="スクリーンショット 2022-05-12 13 30 39" src="https://user-images.githubusercontent.com/100191650/167993811-a4b107be-2c1d-4cfb-b31d-4a522ac27ba0.png">
<img width="1009" alt="スクリーンショット 2022-05-12 13 31 08" src="https://user-images.githubusercontent.com/100191650/167993967-dbf6e381-4ba0-4dc3-aa8e-e81401acfb5a.png">
<img width="1009" alt="スクリーンショット 2022-05-12 13 31 44" src="https://user-images.githubusercontent.com/100191650/167993828-cf3a6fb8-d611-4b00-b704-ef23b1300550.png">
A role with Admin privileges is attached to the stack.This Admin privileges is temporary
(スタックにはAdmin権限を持つロールをアタッチしています。この権限付与は一時的なものです。)
<img width="1009" alt="スクリーンショット 2022-05-12 14 18 31" src="https://user-images.githubusercontent.com/100191650/167997273-b054b752-7364-4d94-9714-647cc985093b.png">

I would be grateful if you could let me know if there are any inadequacies in the settings.
(設定に不備があれば教えていただけると嬉しいです。)

Thank you.

**Comment by yanoty3d** [comment]
Have you solved this? I had exactly the same problem. But I found a solution.

Go to Amazon S3 page and click "Block Public Access settings for this account"(このアカウントのブロックパブリックアクセス設定). Then turn OFF "Block all public access" (パブリックアクセスをすべてブロック)
